frontend/app/templates/create-badges.hbs

```diff
@@ -118,18 +118,11 @@
         </div>
     </div>
     <button {{action "submitForm"}} class="ui fluid button">Create Badges</button>
-    {{#if badgeGenerated}}
```

Comment:
Instead of using timed notification, can't we go with this approach only? Can this variable (badgeGenerated) be set to false whenever the page od reloaded?

Reply:
@djmgit  You are correct but this app is behaving like single page application and page not get refreshed on coming back again to create badges route. We have to override route constructor for setting up the variable false on route reload. I was following the same approach but unable to find the workaround for overriding the route constructor that's why I have done this way. For now, we can go with this. As soon as we find the way we can change the method.
What's your say @djmgit @yashLadha @gabru-md?